packages/node-integration-tests/suites/express/tracing/test.ts

```diff
@@ -53,3 +53,32 @@ test('should set a correct transaction name for routes specified in RegEx', asyn
     },
   });
 });
+
+test.each([
+  ['', 'array1'],
+  ['', 'array5'],
+])('%s should set a correct transaction name for routes consisting of arrays of routes', async (_, segment) => {
```

Comment:
Why do we have `%s` here? Isn't it always going to be an empty string?

Reply:
Ahh good point, will remove it